docker/content-build-2and3/Pipfile

```diff
@@ -21,6 +21,9 @@ ansible-runner = "*"
 rsa = "*"
 paramiko = "*"
 boto3 = "*"
+pyminizip = "*"
+beautifulsoup4 = "*"
+pypdf2 = "*"
```

Comment:
is that case-insensitive?

Reply:
@yaakovi 
Yes its case insensitive, It created by the:
`pipenv install PyPdf2`
[Check it out](https://stackoverflow.com/questions/26503509/is-pypi-case-sensitive)